src/statd/python/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2020 Denis Kalashnikov
```

Comment:
Denis' code was only the basis for `confd`, I see now that we've managed to copy the `confd` LICENSE file to `statd`.  Probably my mistake ... could you help correct that, please?